docs/conf.py

```diff
@@ -49,9 +49,12 @@
 # intersphinx mapping
 intersphinx_mapping = {
     'numpy': ('https://numpy.org/doc/stable/', None),
+    'numpydoc': ('https://numpydoc.readthedocs.io/en/stable/', None),
```

Comment:
This to me seems like a duplicate that can be merged.

Reply:
https://numpydoc.readthedocs.io/en/latest/format.html this page is not in the numpy doc, but we could also just put a hardlink here. maybe we can discuss in the weekly later.